### PR TITLE
feat: add a landing page for the mobile app

### PR DESCRIPTION
**Pull request by raphael0202**
New landing page for mobile app based on @manon-corneille 's work.

It looks like this:
Capture d’écran 2023-09-11 à 15 28 06-fullpage

edit: I've updated the screenshot after the fixes.

### CODE REVIEW

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 1]
```diff
@@ -1,113 +1,213 @@
...
+        <div class="row">
+            <div class="small-12 medium-4 medium-offset-1 column">
+                <h1 class="title-2 emphasized-title">
+                    Choose products that are good for
+                    <span class="highlight-label yellow rotate-left">you</span>
+                    and the
+                    <span class="highlight-label blue rotate-right">planet</span>
+                </h1>
+                <p>The world's largest open food database at your fingertips!</p>
+                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:49]: title=""

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 2]
```diff
@@ -1,113 +1,213 @@
...
+                <a href="https://apps.apple.com/app/open-food-facts/id588797948">
+                <img
+                    src="/images/misc/appstore/black/appstore_US.svg"
+                    alt="Available on the App Store"
+                    width="120"
+                    height="40"
+                ></a>
+            </div>
+            <div class="small-12 medium-6 medium-offset-1 column">
+                <img src="/images/illustrations/scale-ecoscore-nutriscore.svg" alt="scale-nutriscore-ecoscore" style="max-height: 250px">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:65]: better alt if possible

**Comment by g123k** [comment]
Shouldn't we add a link to FDroid?
And maybe also to the Amazon AppStore (similar to the Windows one)

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 3]
```diff
@@ -1,113 +1,213 @@
...
+        <div class="row">
+            <div class="small-12 medium-4 medium-offset-2 column">
+                <img src="/images/misc/app-landing-page/mobile-app-preview/mobile-app-preview_en.svg" alt="Open Food Facts mobile app">
+            </div>
+            <div class="small-12 medium-5 end column">
+                <h1 class="title-2 emphasized-title">
+                    A scan that truly matches
+                    <span class="highlight-label dark-orange rotate-right">who you are</span>
+                </h1>
+                <p class="paragraph-top-margin">From reducing your salt intake to going full vegan, it's easy to set your own personal preferences in the app and start getting smarter recommandations for food.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/open-food-facts-mobile-app.html:85]:
```suggestion
                <p class="paragraph-top-margin">From reducing your salt intake to going full vegan, it's easy to set your own personal preferences in the app and start getting smarter recommendations for food.</p>
```

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 4]
```diff
@@ -1,113 +1,213 @@
...
+                <p>Best of all, the app always warns you if you're about something containing an allergen.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/open-food-facts-mobile-app.html:86]:
```suggestion
                <p>Best of all, the app always warns you if you're about to eat something containing an allergen.</p>
```

**Comment by stephanegigandet** [comment]
@raphael0202 Could you remove the shadows on the rotated boxes? I don't think we use shadows anywhere: https://www.figma.com/file/Qg9URUyrjHgYmnDHXRsTTB/New-website-design-(2022)-(Quentin)?type=design&node-id=169-4043&mode=design

File: lang/en/texts/open-food-facts-mobile-app.html (hunk 1 above)
Comment (raphael0202) [reply lang/en/texts/open-food-facts-mobile-app.html:49]: ?

Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:49]:
```suggestion
                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en" title="Open Food Facts on Google Play">
```

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 5]
```diff
@@ -1,113 +1,213 @@
...
+                </h1>
+                <p>The world's largest open food database at your fingertips!</p>
+                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">
+                <img
+                    src="/images/misc/playstore/img/en_get.svg"
+                    alt="Available on Google Play"
+                    width="120"
+                    height="40"
+                ></a>
+                <a href="https://apps.apple.com/app/open-food-facts/id588797948">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:56]:
```suggestion
                <a href="https://apps.apple.com/app/open-food-facts/id588797948" title="Open Food Facts on the Apple App Store">
```

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 6]
```diff
@@ -1,113 +1,213 @@
...
+                ></a>
+            </div>
+            <div class="small-12 medium-6 medium-offset-1 column">
+                <img src="/images/illustrations/scale-ecoscore-nutriscore.svg" alt="A scale with a Nutri-Score and a smiley face on the left side of the scale and an Eco Score and planet Earth on the right side" style="max-height: 250px">
+            </div>
+        </div>
+        <div class="row show-for-large-up" style="padding-top: 4rem; text-align: center">
+            <div class="small-3 small-centered column">
+                <p>Not on your mobile? Scan this QR code from your mobile to download the app!</p>
+                <img src="/images/misc/app-landing-page/qrcode_en.svg" width="180" height="180">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:71]:
```suggestion
                <img src="/images/misc/app-landing-page/qrcode_en.svg" width="180" height="180" alt="QR-Code linking to https://world.openfoodfacts.org/open-food-facts-mobile-app">
```

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 7]
```diff
@@ -1,113 +1,213 @@
...
+                        loading="lazy"
+                        alt="Rating on Google Play Store and Apple Store"
+                        loading="lazy"
+                    >
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="small-12 medium-7 medium-offset-5 column">
+                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:172]:
```suggestion
                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en" title="Open Food Facts on Google Play">
```

File: lang/en/texts/open-food-facts-mobile-app.html [hunk 8]
```diff
@@ -1,113 +1,213 @@
...
+                <img
+                    src="/images/misc/playstore/img/en_get.svg"
+                    loading="lazy"
+                    alt="Available on Google Play"
+                    width="120"
+                    height="40"
+                ></a>
+                <a href="https://f-droid.org/packages/openfoodfacts.github.scrachx.openfood">
```
Comment (teolemon) [inline lang/en/texts/open-food-facts-mobile-app.html:180]:
```suggestion
                <a href="https://f-droid.org/packages/openfoodfacts.github.scrachx.openfood" title="Open Food Facts on F-Droid">
```

File: lang/en/texts/open-food-facts-mobile-app.html (hunk 1 above)
Comment (raphael0202) [reply lang/en/texts/open-food-facts-mobile-app.html:49]: is it something usually advised for accessibility?

Comment (teolemon) [reply lang/en/texts/open-food-facts-mobile-app.html]: no idea: https://www.w3.org/WAI/WCAG22/Techniques/html/H33.html

**Review by teolemon — APPROVED** [review]

**Pull request merged**